tests/dsl.test.ts

```diff
@@ -3,15 +3,8 @@ import { parseDSL } from "../src/parse-dsl";
 
 describe("DSL", () => {
   describe("parseDSL()", () => {
```

Comment:
add a test for a DSL that has comments? 😄

Reply:
Right now I'd consider that out of scope just because we don't support comments yet. I'll work on improving support for them (in the parser at least) in an upcoming PR